pinc/base.inc

```diff
@@ -23,6 +23,10 @@ if (!headers_sent()) {
     // Disallow other sites from embedding pages in frames/iframes
     header("Content-Security-Policy: frame-ancestors 'self'", false);
     header("X-Frame-Options: SAMEORIGIN");
+    header("X-Content-Type-Options: nosniff");
+    if ($use_secure_cookies) {
+      header("Strict-Transport-Security: max-age=63072000; includeSubDomains; preload");
```

Comment:
While pgdp.net does HTTP to HTTPS redirect, not all sites that use the code may. This would be better served in the `pgdp-production` branch that is just for pgdp.net.

Reply:
This also needs 2 more spaces indent.

Reply:
I removed these. We can create a separate PR for prod hacks and investigate nosniff separately.